examples/expect-extend/toBeWithinRange.ts

```diff
@@ -5,8 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {expect} from '@jest/globals';
-import type {MatcherFunction} from 'expect';
+import {type MatcherFunction, expect} from 'expect';
```

Comment:
What about this? If `'expect'` is extended later, perhaps importing from `'expect'` here makes sense? Or?

Reply:
agreed!

Reply:
at least, I agree if this example is "extend expect" and not "extend jest with extra matcher". In that case people should import from `@jest/globals` (or use the global whenever we fix that)

Reply:
Hm.. Perhaps I will revert this. In my head this was "extend jest with extra matcher" example. I mean, there is an extra file where this new matcher is used in a test. If `expect` was extended not for Jest, then `expect(100).toBeWithinRange(90, 110);` line in this file would be enough without extra test file. It makes sense to import from `'@jest/globals'`.

Reply:
👍 

We should work on https://jestjs.io/docs/expect to give better examples there probably, but keep the example as is

Reply:
Agreed (;